pkg/docker/registry/registry.go

```diff
@@ -53,7 +53,34 @@ var secretAnnotations = map[string]string{
 	"helm.sh/hook-weight": "-9999",
 }
 
-func ProxyEndpointFromLicense(license *kotsv1beta1.License) *RegistryProxyInfo {
+func GetRegistryProxyInfo(license *kotsv1beta1.License, app *kotsv1beta1.Application) *RegistryProxyInfo {
+	registryProxyInfo := getRegistryProxyInfoFromLicense(license)
+	proxyEndpoint, _ := getRegistryProxyEndpointFromKotsApplication(app)
```

Comment:
How come this ignores the second value?

Reply:
I had to revert 2 commits https://github.com/replicatedhq/kots/pull/3447/commits/713520bed4cb331f9f5472a15c0c6ec1b8fc97d1 https://github.com/replicatedhq/kots/pull/3447/commits/5e530c49451192167d1a70a2aaf81c43a6d04c9f relavent to registryEndpoint which will be covered in another SC https://app.shortcut.com/replicated/story/62143

Reply:
ok, let's review this again when everything is merged.  there are already some conflicts here